backend/Cell.py

```diff
@@ -1,7 +1,7 @@
 from enum import Enum
 
 
-class CellType(Enum):
+class Color(Enum):
```

Comment:
I do have somewhat mixed feelings about this, since the turn shouldn't ever be listed as VOID or EMPTY and now has the potential ability to be either... but ultimately we're handling turn swapping without the user being involved, so I suppose it doesn't matter. 

The refactor here looks good. Thank you.